packages/patterns/src/components/partial-syncing-controls.js

```diff
@@ -71,25 +72,21 @@ function PartialSyncingControls( { name, attributes, setAttributes } ) {
 		<InspectorControls group="advanced">
 			<BaseControl __nextHasNoMarginBottom>
 				<BaseControl.VisualLabel>
-					{ __( 'Synced attributes' ) }
+					{ __( 'Pattern overrides' ) }
 				</BaseControl.VisualLabel>
-				{ Object.entries( syncedAttributes ).map(
-					( [ attributeName, label ] ) => (
-						<CheckboxControl
-							key={ attributeName }
-							__nextHasNoMarginBottom
-							label={ label }
-							checked={
-								attributes.connections?.attributes?.[
-									attributeName
-								]?.source === 'pattern_attributes'
-							}
-							onChange={ ( isChecked ) => {
-								updateConnections( attributeName, isChecked );
-							} }
-						/>
-					)
-				) }
+				<CheckboxControl
+					__nextHasNoMarginBottom
+					label={ __( 'Allow instance overrides' ) }
+					checked={ Object.keys( syncedAttributes ).some(
+						( attributeName ) =>
+							attributes.connections?.attributes?.[
+								attributeName
+							]?.source === 'pattern_attributes'
+					) }
+					onChange={ ( isChecked ) => {
+						updateConnections( isChecked );
+					} }
+				/>
```

Comment:
I think would also be good to disable the checkbox when the block's only connectable attributes are connected to something else (e.g. a custom field).

At the moment, if you try checking it nothing happens.

Reply:
I wonder if we should just completely avoid rendering it if that happens. Disabling it might confuse the users too as we don't show any other helper text. Completely hiding it has the same issue but I think it's easier 😅. Given that both features (meta binding and pattern overrides) are not stable yet, I think it's best to keep it simple here for now. WDYT?

Reply:
Yeah, I wasn't sure, but I'm good with either 👍 

I'll make sure to add it to the tracking issue as something to reconsider.